src/os/bluestore/BlueStore.cc

```diff
@@ -16767,8 +16784,13 @@ uint8_t RocksDBBlueFSVolumeSelector::select_prefer_bdev(void* h) {
 
 void RocksDBBlueFSVolumeSelector::get_paths(const std::string& base, paths& res) const
 {
-  res.emplace_back(base, l_totals[LEVEL_DB - LEVEL_FIRST]);
-  res.emplace_back(base + ".slow", l_totals[LEVEL_SLOW - LEVEL_FIRST]);
+  auto db_size = l_totals[LEVEL_DB - LEVEL_FIRST];
+  res.emplace_back(base, db_size);
+  auto slow_size = l_totals[LEVEL_SLOW - LEVEL_FIRST];
+  if (slow_size == 0) {
+    slow_size = db_size;
+  }
+  res.emplace_back(base + ".slow", slow_size);
```

Comment:
@ifed01 Could you please remind me, why we must provide base + ".slow" to RocksDB ?
I mean - cant we just provide base ?

Reply:
Originally we provide db.slow vs.  db for RocksDB to be able to indicate  less frequently used data. Hence we're able to put them to larger but slower main device. 
This works fine for 2+ device setups.
For single device setup we had that "dual-folder" provisioning in RocksDBBlueFSVolumeSelector from its introduction but it wasn't used since that selector isn't created for single device setup at all.
For OriginalVolumeSelector in https://github.com/ceph/ceph/pull/42992 I introduced dual folder provisioning for single device setup as well recently - to fix the case when files originally put into db.slow are still referred via this subdir after DB volume migration/removal.
So now we have unconditional dual-folder setup for both selectors.
And it's still a no-op for RocksDBBlueFSVolumeSelector as the latter is not used for single-device deployment for now. 
Perhaps we can change this some day though, primarily for the sake of getting rid of the original selector and having a bit better bluefs stats reporting..
So while preparing this patch (which is primarily for original selector) I additionally tried RocksDBBlueFSVolumeSelector in a single device setup and discovered that it should adjust indicated capacity for slow folder for RocksDB to parse that properly. The latter was complaining about size set to 0...
So despite this specific modification in RocksDBBlueFSVolumeSelector::get_path() is generally unrelated to the proposed fix I kept it as-is for the sake of future selector unification.

Reply:
@aclamk - ping